### PR TITLE
RIV-971 updated insights reference documentation

### PR DESCRIPTION
**Pull request by wsagen**

### CODE REVIEW

File: dev-shipperhq/static/order-view-service/index.html [hunk 1]
```diff
@@ -3,7 +3,7 @@
   <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>ShipperHQ Insight API Static Documentation</title>
+    <title>ShipperHQ Shipping Insights API Static Documentation</title>
```
Comment (bligneri) [inline dev-shipperhq/static/order-view-service/index.html:6]: to remove

**Review by bligneri — APPROVED** [review]
Did modify the generated file so that we remove `Shipping` to only have `Insights API`
Modify both insights and rates to point to the support page (vs support@shipperhq.com)

**Comment by bligneri** [comment]
I just modified the spectaql config files so that we can re-generate the doc and have:
* Uniform title and document names
* Uniform support links
* Uniform back-links to the dev portal ($API_NAME guide)

**Review by wsadaniel — APPROVED** [review]
Fixed a few capitalization/text consistency issues on config and built files.

**Pull request merged**